app/adapters/order.js

```diff
@@ -0,0 +1,11 @@
+import ApplicationAdapter from './application';
+
+export default ApplicationAdapter.extend({
+  buildURL(modelName, id, snapshot, requestType, query) {
```

Comment:
Code repetition ... Add this to the main adapter to ensure this functionality is available for all models. Also remove from event adapter.